src/upload/imgurprovider.h

```diff
@@ -0,0 +1,47 @@
+/*
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) 2013  PCMan <email>
```

Comment:
Don't you want to gain the credits intentionally or is it just a copy/paste?

Reply:
I looked at the other headers and wondered the same thing :smile: I'd be more than happy to put my name on the new files if that's not a problem.